x-pack/plugins/ml/public/ui_actions/edit_explorer_panel_action.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { createAction } from '../../../../../src/plugins/ui_actions/public';
+import { ViewMode } from '../../../../../src/plugins/embeddable/public';
+import { MlCoreSetup } from '../plugin';
+import { ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE, EditExplorerPanelContext } from '../embeddables';
+
+export const EDIT_EXPLORER_PANEL_ACTION = 'editExplorerPanelAction';
+
+export function createEditExplorerPanelAction(getStartServices: MlCoreSetup['getStartServices']) {
+  return createAction<EditExplorerPanelContext>({
+    id: 'edit-anomaly-explorer',
+    type: EDIT_EXPLORER_PANEL_ACTION,
+    getIconType(context): string {
+      return 'pencil';
+    },
+    getDisplayName: () =>
+      i18n.translate('xpack.ml.actions.editExplorerTitle', {
+        defaultMessage: 'Edit explorer',
```

Comment:
Suggest changing the text here to `Edit charts` for now.

Reply:
Renamed here [`3c31e37` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/3c31e37dacc795ad9e4b0b71d459616bda58b979)